04/global.css

```diff
@@ -1,2 +1,60 @@
 /*font-family: Georgia, Times, "Times New Roman", serif;
-font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;*/
\ No newline at end of file
+font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;*/
+#header{
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 27px;
+    padding-left: 91px;
+    padding-top: 21px;
+    padding-bottom: 20px;
+    border-bottom: 1px;
+    border-bottom-color: orange;
+    border-bottom-style: solid;
+    border-bottom-width: thin;
+    font-weight: bolder;
+    letter-spacing: 3px;
+    color: orange;
+}
+
+#header_description {
+      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+            padding: 20px;
+    margin: 20px;
+    background-color: #F2F2F2;
+        letter-spacing: .5px;
+}
+#ingredient_header{
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+ 
+    padding-left: 20px;
+    font-size: 22px;
+    text-decoration: underline;
+    margin-top: 20px;
+    font-weight: bold;
+  font-family: Georgia, Times, "Times New Roman", serif;
+font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+#ingredients{
+      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    padding-top: 9px;
+    padding-left: 40px;
+    list-style-type: square;
+  font-family: Georgia, Times, "Times New Roman", serif;
+font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+#method_header{
+      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 20px;
+    letter-spacing: 2px;
+    padding-top: 30px;
+    padding-left: 20px;
+    text-decoration: underline;
+}
+#method{
+          font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 18px;
+    padding-top: 10px;
+    padding-left: 40px;
+    letter-spacing: -1px;
+        list-style-type: decimal;
+}
```

Comment:
Consider how you might DRY up some of this code. Would using classes help, since you can apply CSS to a class (e.g. `.thing { font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; }`) and then apply that class to many elements in the HTML?